itests/kit/evm.go

```diff
@@ -310,3 +324,30 @@ func removeLeadingZeros(data []byte) []byte {
 	}
 	return data[firstNonZeroIndex:]
 }
+
+func SetupFEVMTest(t *testing.T) (context.Context, context.CancelFunc, *TestFullNode) {
+	//make all logs extra quiet for fevm tests
+	lvl, err := logging.LevelFromString("error")
+	if err != nil {
+		panic(err)
+	}
+	logging.SetAllLoggers(lvl)
+
+	blockTime := 100 * time.Millisecond
+	client, _, ens := EnsembleMinimal(t, MockProofs(), ThroughRPC())
+	ens.InterconnectAll().BeginMining(blockTime)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+
+	// instead of having this in a few tests put this in the setup
+	// verify that the initial balance is 100000000000000000000000000
+	// this allows other tests to assert the initial balance
+	// without explicitly checking it
+	fromAddr := client.DefaultKey.Address
+	bal, err := client.WalletBalance(ctx, fromAddr)
+	require.NoError(t, err)
+	originalBalance, err := big.FromString("100000000000000000000000000")
```

Comment:
You can use `types.FromFil()` to convert from Fil to attofil. This number is too hard to read otherwise.

Reply:
https://github.com/filecoin-project/lotus/pull/10082/files#diff-a6568b1d9d63044a907e923153b083711bc137853ad6a9d85e0a4ade8b5d6ffbR341